app/javascript/packs/mga_app.ts

```diff
@@ -0,0 +1,39 @@
+import Vue from 'vue'
```

Comment:
This file is only temporary I guess, and can be removed (as all other mga-related changes?)?

Reply:
Yep